docs/available-filters/record-encryption/aws-kms/creating_keys.adoc

```diff
@@ -0,0 +1,27 @@
+:aws:  https://docs.aws.amazon.com/
+
+= AWS Key Management Service
+
+As the Administrator, use either the AWS Console or CLI to
+{aws}/kms/latest/developerguide/create-keys.html#create-symmetric-cmk[create] a *Symmetric key* with *Encrypt and decrypt*
+usage.  Multi-region keys are supported.  It is not possible to make use of keys from other AWS accounts.
+
+Give the key an alias following the link:setup.adoc#_establish_an_aliasing_convention_for_keys_within_aws_kms[alias naming convention] established above.
+
+If using the CLI, this can be done with commands like this:
+
+[source,shell]
+----
+KEY_ID=$(aws kms create-key | jq -r '.KeyMetadata.KeyId')
+# the create key command will produce JSON output including the KeyId
+aws kms create-alias --alias-name alias/KEK_<name> --target-key-id ${KEY_ID}
+----
+
+Once the key is created, it is recommended to use a key rotation policy.
```

Comment:
I think it would be worth clarifying that this is a KEK rotation policy and doesn't affect the DEK's the proxy actually encrypts records with.

Reply:
We already have general content that talks about rotation. https://kroxylicious.io/kroxylicious/#_key_rotation.  I don't think there's anything AWS specific we need to call out here.